neo/VM/OpCode.py

```diff
@@ -207,17 +203,22 @@
 KEYS = b'\xCC'
 VALUES = b'\xCD'
 
+# Stack isolation
```

Comment:
Would be nice to add in some comments on these opcodes from here: https://github.com/neo-project/proposals/pull/22/files#diff-1d482db713cb2319d6501a7c910c110aR29

Reply:
will do

Reply:
will do